app/scripts/controllers/permissions/specifications.test.js

```diff
@@ -15,10 +15,13 @@ describe('PermissionController specifications', () => {
   describe('caveat specifications', () => {
     it('getCaveatSpecifications returns the expected specifications object', () => {
       const caveatSpecifications = getCaveatSpecifications({});
-      expect(Object.keys(caveatSpecifications)).toHaveLength(1);
+      expect(Object.keys(caveatSpecifications)).toHaveLength(2);
       expect(
         caveatSpecifications[CaveatTypes.restrictReturnedAccounts].type,
       ).toStrictEqual(CaveatTypes.restrictReturnedAccounts);
+      expect(caveatSpecifications.permittedDerivationPaths.type).toStrictEqual(
```

Comment:
Can you add this to the `CaveatTypes` enum similar to how it is for `restrictReturnedAccounts`?

Reply:
It's in `SnapCaveatTypes` in the `rpc-methods` package, but that isn't exported at the moment. We can change it in the future if we export that?

Reply:
do you mind leaving a comment in the code for that todo?